### PR TITLE
Fix `release` for Windows and updated test to use Windows AU.  

### PR DESCRIPTION
**Pull request by michaeljota**

### CODE REVIEW

File: is.js [hunk 1]
```diff
@@ -106,7 +106,7 @@ IsApi.prototype.release = function (requested) {
   if (this.osx()) {
     return requested === osxRelease()
   } else if (this.windows()) {
-    let requested = requested.split('.')
+    requested = requested.split('.')
```
Comment (delvedor) [inline is.js]: Can you do the change you did here in gtRelease and ltRelease too?

Comment (michaeljota) [inline is.js]: Oh! I did not see those. Of curse.

Comment (michaeljota) [inline is.js]: Done. :)

**Comment by delvedor** [comment]
Looks good to me, thanks for contributing!
In the future I must implement something that manages the OS version test automatically, without hardcoding it :D

**Pull request merged**